app/src/main/java/test1/daihocbachkhoa/tranphuc1995/assignment1/MainActivity.java

```diff
@@ -0,0 +1,51 @@
+package test1.daihocbachkhoa.tranphuc1995.assignment1;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+
+    RecyclerView recyclerView;
```

Comment:
Why these fields are declared in `package` scope? Do you know about different scopes?